deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/worker/BuildJanitor.java

```diff
@@ -39,6 +44,8 @@
 public class BuildJanitor implements Job {
```

Comment:
Just learned that we have worker implementing the `Job` interface. We might want to refactor them in the future.

Reply:
ok